zegami_sdk/collection.py

```diff
@@ -1020,6 +1025,35 @@ def __len__(self) -> int:
     def __repr__(self) -> str:
         return "<Collection id={} name={}>".format(self.id, self.name)
 
+    def id_to_class(self, ID):
```

Comment:
I'm not sure what this method is targeted at, uploaded classes are 0-indexed. Is this method referring to 1-indexed classes in a specific project? Typical class specification in a collection is:
```
[{'color': '#e6b3b3', 'id': '0', 'name': 'Damage'},
 {'color': '#bfb340', 'id': '1', 'name': 'Corrosion'},
 {'color': '#000000', 'id': '2', 'name': 'Dummy0'},
 {'color': '#000000', 'id': '3', 'name': 'Dummy1'},
 {'color': '#000000', 'id': '4', 'name': 'Dummy2'}]
```